assets/training/model_management/src/azureml/model/mgmt/processors/transformers/convertors.py

```diff
@@ -160,33 +163,108 @@ def _save(
+            pip_requirements = None
+
+        # handle OSS trust_remote_code value
+        trust_remote_code_val = False
+        config_args = self._hf_conf.get(HF_CONF.HF_CONFIG_ARGS.value, {})
+        tokenizer_args = self._hf_conf.get(HF_CONF.HF_TOKENIZER_ARGS.value, {})
+        model_args = self._hf_conf.get(HF_CONF.HF_MODEL_ARGS.value, {})
+
+        if (config_args.get('trust_remote_code', False)
+                and tokenizer_args.get('trust_remote_code', False)
+                and model_args.get('trust_remote_code', False)):
+            trust_remote_code_val = True
+
+        model_pipeline = transformers.pipeline(task=self._task, model=model,
+                                               trust_remote_code=trust_remote_code_val)
+
+        params = {
+            "transformers_model": model_pipeline,
+            "code_paths": code_paths,
+            "signature": self._signatures,
+            "input_example": input_example,
+            "metadata": metadata,
+            "path": str(self._output_dir),
+        }
+
+        if curated_conda_env:
+            params.update({
+                "conda_env": curated_conda_env,
+            })
+        elif pip_requirements:
+            params.update({
+                "pip_requirements": pip_requirements,
+            })
+        else:
+            params.update({
+                "extra_pip_requirements": self._extra_pip_requirements,
+            })
+
+        mlflow.transformers.save_model(**params)
+
+        logger.info("Model saved with mlflow OSS flow for task: {}".format(self._task))
+
+    def _save_in_hftransformersv2_flavor(self, metadata, conda_env,
+                                         code_paths, input_example,
+                                         requirements_file, pip_requirements):
 
         # Set experimental flag
         if self._experimental:
             logger.info("Experimental features enabled for MLflow conversion")
             self._hf_conf["exp"] = True
 
-        # set metadata info
-        metadata = fetch_mlflow_acft_metadata(base_model_name=self._model_id,
-                                              is_finetuned_model=False,
-                                              base_model_task=self._task)
+        logger.info("Segregate input model dir and present into separate folders for model, config and tokenizer")
+        logger.info("Preparing model files")
+        tmp_model_dir = Path(self._temp_dir) / HF_CONF.HF_MODEL_PATH.value
+        copy_files(self._model_dir, tmp_model_dir)
+        model = str(tmp_model_dir)
+        logger.info("Loading config")
+        config = self._hf_config_cls.from_pretrained(
+            self._model_dir, **self._hf_conf.get(HF_CONF.HF_CONFIG_ARGS.value, {})
+        )
+        logger.info("Loading tokenizer")
+        tokenizer = self._hf_tokenizer_cls.from_pretrained(
+            self._model_dir, **self._hf_conf.get(HF_CONF.HF_TOKENIZER_ARGS.value, {})
+        )
+
```

Comment:
setting signature for hftv2 might be an issue. Could you please confirm once that its not set?